qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -0,0 +1,431 @@
+                that takes an array as input and returns a Qiskit or SciPy optimization result.
+            k (int): the number of eigenvalues to return. Returns the lowest k eigenvalues.
+            betas (list[float]): beta parameters in the VQD paper.
+                Should have length k - 1, with k the number of excited states.
+                These hyper-parameters balance the contribution of each overlap term to the cost
+                function and have a default value computed as the mean square sum of the
+                coefficients of the observable.
+            callback (Callable[[int, np.ndarray, float, dict[str, Any]], None] | None):
+                a callback that can access the intermediate data
+                during the optimization. Four parameter values are passed to the callback as
+                follows during each evaluation by the optimizer: the evaluation count,
+                the optimizer parameters for the ansatz, the evaluated mean, the evaluation
+                metadata, and the current step.
+    """
+
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        fidelity: BaseStateFidelity,
+        ansatz: QuantumCircuit,
+        optimizer: Optimizer | Minimizer,
+        *,
+        k: int = 2,
+        betas: list[float] | None = None,
+        initial_point: Sequence[float] | None = None,
+        callback: Callable[[int, np.ndarray, float, dict[str, Any]], None] | None = None,
+    ) -> None:
+        """
+
+        Args:
+            estimator: The estimator primitive.
+            fidelity: The fidelity class using primitives.
+            ansatz: A parameterized circuit used as ansatz for the wave function.
+            optimizer: A classical optimizer. Can either be a Qiskit optimizer or a callable
+                that takes an array as input and returns a Qiskit or SciPy optimization result.
+            k: the number of eigenvalues to return. Returns the lowest k eigenvalues.
+            betas: beta parameters in the VQD paper.
+                Should have length k - 1, with k the number of excited states.
+                These hyperparameters balance the contribution of each overlap term to the cost
+                function and have a default value computed as the mean square sum of the
+                coefficients of the observable.
+            initial_point: an optional initial point (i.e. initial parameter values)
+                for the optimizer. If ``None`` then VQD will look to the ansatz for a preferred
+                point and if not will simply compute a random one.
+            callback: a callback that can access the intermediate data
+                during the optimization. Four parameter values are passed to the callback as
+                follows during each evaluation by the optimizer: the evaluation count,
+                the optimizer parameters for the ansatz, the evaluated mean, the evaluation
```

Comment:
In VQE the wording was changed https://github.com/Qiskit/qiskit-terra/blob/293ab43afb87a75ffe4157fa629830afa42a5fe4/qiskit/algorithms/minimum_eigensolvers/vqe.py#L137-L138

It used to say mean there to but we went with `estimated value` since that seemed more relevant to the estimator return which says nothing about mean right.